dbus-opendtu.py

```diff
@@ -80,6 +80,9 @@ def main():
             "/Ac/L1/Energy/Forward": {"initial": None, "textformat": _kwh},
             "/Ac/L2/Energy/Forward": {"initial": None, "textformat": _kwh},
             "/Ac/L3/Energy/Forward": {"initial": None, "textformat": _kwh},
+            "/Ac/Out/L1/I": {"initial": None, "textformat": _a},
```

Comment:
Fixed L1? Shouldn't this use the config?

Reply:
According to [Victrons list](https://github.com/victronenergy/dbus_modbustcp/blob/master/CCGX-Modbus-TCP-register-list.xlsx) there's only L1 in the path. Maybe they don't sell any 3 Phase inverters.

Reply:
Ok. Understand. Maybe this is similar to the Multiplus who also always assumes that its grid connection is L1.